docs/dev/documentation.rst

```diff
@@ -0,0 +1,96 @@
+Documentation
+=============
+
+.. _building-the-docs:
+
+Building the docs
+-----------------
+
+Dependencies for building the documentation for scanpy can be installed with ``pip install -e "scanpy[doc]"``
+
+To build the docs, enter the ``docs`` directory and run `make html`. After this process completes you can take a look at the docs by opening ``scanpy/docs/_build/html/index.html``.
```

Comment:
We have single-backticks correspond to "code" styling. So, anything that you have with double backticks could be single. Just a suggestion, no need to change.